src/schema/document.rs

```diff
@@ -155,6 +155,23 @@ impl Document {
             .find(|field_value| field_value.field() == field)
             .map(FieldValue::value)
     }
+
+    /// Prepares Document for being stored in the document store
+    ///
+    /// Method transforms PreTokenizedString values into String
+    /// values.
+    pub fn prepare_for_store(&mut self) {
+        let mut i = 0;
+        while i < self.field_values.len() {
```

Comment:
why not a for-loop?

Reply:
No particular reason. I'll change to `for` if that's the preferred way.

Reply:
Yes please use a for loop whenever possible. It makes it easier to proofread.